packages/esm-patient-notes-app/src/notes/ward-in-patient/notes-form.component.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import dayjs from 'dayjs';
+import { useTranslation } from 'react-i18next';
+import { z } from 'zod';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { Controller, useForm } from 'react-hook-form';
+import { Button, Column, Form, InlineLoading, Row, Stack, TextArea } from '@carbon/react';
+import {
+  createErrorHandler,
+  ResponsiveWrapper,
+  showSnackbar,
+  translateFrom,
+  useConfig,
+  useSession,
+} from '@openmrs/esm-framework';
+import { type DefaultPatientWorkspaceProps } from '@openmrs/esm-patient-common-lib';
+import type { ConfigObject } from '../../config-schema';
+import { saveVisitNote } from '../visit-notes.resource';
+import styles from './notes-form.scss';
+import { moduleName } from '@openmrs/esm-patient-medications-app/src/dashboard.meta';
+
+type NotesFormData = z.infer<typeof noteFormSchema>;
+
+const noteFormSchema = z.object({
+  wardClinicalNote: z.string().refine((val) => val.trim().length > 0, {
+    message: translateFrom(moduleName, 'clinicalNoteErrorMessage', 'Clinical note is required'),
```

Comment:
Add a comment above this line
```suggestion
		//t('clinicalNoteErrorMessage','Clinical note is required')
    message: translateFrom(moduleName, 'clinicalNoteErrorMessage', 'Clinical note is required'),
```